Kinvey.Core/Realtime/RealtimeReconnectionPolicy.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) 2018, Kinvey, Inc. All rights reserved.
+//
+// This software is licensed to you under the Kinvey terms of service located at
+// http://www.kinvey.com/terms-of-use. By downloading, accessing and/or using this
+// software, you hereby accept such terms of service  (and any agreement referenced
+// therein) and agree that you have read, understand and agree to be bound by such
+// terms of service and are of legal age to agree to such terms with Kinvey.
+//
+// This software contains valuable confidential and proprietary information of
+// KINVEY, INC and is subject to applicable licensing agreements.
+// Unauthorized reproduction, transmission or distribution of this file and its
+// contents is a violation of applicable laws.
+
+namespace Kinvey
+{
+    /// <summary>
+	/// Enumeration for realtime reconnection policy
+	/// </summary>
+    public enum RealtimeReconnectionPolicy
+    {
```

Comment:
For `enum` values in C#, are these normally in all capital letters (i.e. `NONE` rather than `None`)? For example, the Pubnub enums for these are all capitalized.

Reply:
@vinaygahlawat 
I have found a guideline https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions
According to it enum values need to have a PascalCasing in naming.

But I had a look at our code. We use enum values with capital letters almost everywhere.
I  have taken it into consideration and have changed "RealtimeReconnectionPolicy" enum values to capital letters style.

Reply:
This is a good find, thanks! Let's make the change to all caps here for consistency, but I'll create a ticket to move all of our enums to this convention in the future, and we can do it everywhere in one PR.